sdk/python/feast/infra/online_stores/contrib/cassandra_online_store/cassandra_online_store.py

```diff
@@ -88,7 +88,7 @@
         event_ts        TIMESTAMP,
         created_ts      TIMESTAMP,
         PRIMARY KEY ((entity_key), feature_name)
-    ) WITH CLUSTERING ORDER BY (feature_name ASC);
+    ) WITH CLUSTERING ORDER BY (feature_name ASC) {optional_ttl_clause};
```

Comment:
We can change to `WITH default_time_to_live = {ttl}`

Reply:
Will handle ttl in a separate PR.